utils/src/test/java/com/cloud/utils/HumanReadableJsonTest.java

```diff
@@ -50,4 +51,14 @@ public void parseMatchJsonTest() {
         assertEquals("[{\"size\":\"(0 bytes) 0\"}]", getHumanReadableBytesJson("[{\"size\": \"0\"}]"));
         assertEquals("[{\"size\":\"(0 bytes) 0\",\"bytesSent\":\"(0 bytes) 0\"}]", getHumanReadableBytesJson("[{\"size\": \"0\", \"bytesSent\": \"0\"}]"));
     }
+
+    @Test
+    public void localeTest() {
+        Locale.setDefault(Locale.UK); // UK test
```

Comment:
@Spaceman1984 what's the default locale for CloudStack, is there a global setting or does it pick it up from OS?

Reply:
@rhtyd From Java Docs:
"The Java Virtual Machine sets the default locale during startup based on the host environment. It is used by many locale-sensitive methods if no locale is explicitly specified. It can be changed using the setDefault method."

Reply:
I think "C" is the default locale. maybe this or some other more worldly locale is a good idea?

Reply:
@DaanHoogland I searched the code base, Cloudstack doesn't set a default locale, it will be picked up from the environment, which I believe is the correct behaviour.